### DISCUSSION TITLE
Table valued parameters with SqlBuilder

### DISCUSSION THREAD

**Discussion by dplaskon** [maxtoroq/DbExtensions#34] [Q&A] [answered]
Is it possible to use Table valued parameters (https://msdn.microsoft.com/en-us/library/bb675163%28v=vs.110%29.aspx) with SqlBuilder? If so, how?

**Comment by maxtoroq** [comment, marked as answer]
You can use any object as parameter value, but if the parameter needs special configuration you'd need to create the command, make your changes then execute it.

**Comment by dplaskon** [comment]
That's the route I ended up taking - thanks!
